src/components/transport_manager/include/transport_manager/usb/libusb/usb_handler.h

```diff
@@ -36,7 +36,7 @@
 #ifndef SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_USB_LIBUSB_USB_HANDLER_H_
 #define SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_USB_LIBUSB_USB_HANDLER_H_
 
-#include <libusb/libusb.h>
+#include "libusb/libusb.h"
```

Comment:
The rule is old as the world: system includes should be in `<>` and all other in `" "`